code/implementation/docker/machine/src/main/resources/schema/base/rackspaceConfig.json.d/rackspace.json

```diff
@@ -0,0 +1,42 @@
+{
+    "resourceFields": {
+        "username": {
+            "type": "string",
+            "nullable": false,
+            "required": true
+        },
+        "apiKey": {
+            "type": "string",
+            "nullable": false,
+            "required": true
+        },
+        "region": {
+            "type": "string",
+            "nullable": true
+        },
+        "endpointType": {
+            "type": "string",
+            "nullable": true
+        },
+        "imageId": {
+            "type": "string",
+            "nullable": true
+        },
+        "flavorId": {
+            "type": "string",
+            "nullable": true
+        },
+        "sshUser": {
+            "type": "string",
+            "nullable": true
+        },
+        "sshPort": {
+            "type": "string",
+            "nullable": true
+        },
+        "dockerInstall": {
```

Comment:
It's less-than-ideal that we're using a string type where the value is going to be "true", but it looks like in the cli, they treat it as a string and want `--docker-install "true"`vs just `--docker-install` like we've seen with other truly boolean parameters.
@wlan0 have you tested this one specifically? Just asking because we got burnt by booleans once before.